### DISCUSSION TITLE
Boleto Banco Daycoval

### DISCUSSION THREAD

**Discussion by alessandrobrunolima** [braully/bopepo#22] [Q&A] [answered]
Bom dia, estou precisando gerar boleto para o Banco Daycoval, gostaria de saber se alguém já implementou algo?

**Comment by braully** [comment, marked as answer]
Sim, o pessoal do devcoffee desenvolveu para o fork deles do bopepo: 
- https://github.com/devcoffee/bopepo/
- https://github.com/devcoffee/bopepo/blob/0.2-Helio/src/main/java/org/jrimum/bopepo/view/info/campo/BoletoInfoViewDaycoval.java

Já tem um tempo que estou tentando incorporar diversos novidades de outros forks, inclusive essa do banco daycoval, tenho uma branch separado para isso, está funcionando, porém preciso fazer mais testes para liberar em versão final, se etiver interesse em:  testar está em: https://github.com/braully/bopepo/tree/merge-melhorias-terceiros
